mitiq/calibration/tests/test_calibration.py

```diff
@@ -119,6 +120,27 @@ def test_ZNE_workflow():
     assert isinstance(cal.best_strategy(), Strategy)
 
 
+def pec_execute(circuit, noise_level=0.01):
```

Comment:
Can we rename this to `damping_execute`?  The term "pec" in the name is a bit confusing. 
The difference with the existing `execute` is that it simulates amplitude damping noise instead of depolarizing. 

Actually, given that we assume depolarizing noise in `PECSettings`, why do we need to simulate amplitude damping noise? Can't we just use the existing `execute` function?

Reply:
I've fixed the naming to indicate the types of noise used. ZNE tests were already set up with amplitude damping noise, but we cannot "represent with amplitude damping" on non-Cirq frontends for PEC, so I have to stick with depolarizing.